float64[2] dipan_v
